portal-web/test/functional/com/liferay/portalweb/macros/User.macro

```diff
@@ -1418,6 +1418,40 @@ definition {
 			userScreenName = "${userScreenName}");
 	}
 
+	macro editPasswordCPWithoutLogin {
```

Comment:
Replace by using JSONUser.setFirstPassword

Reply:
I remove this method